.yarnrc.yml

```
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
```

```suggestion
nodeLinker: "pnp"
```

요렇게 바꾸면 pnp 모드로 동작해요~!